app/views/splash/index.html.erb

```diff
@@ -39,6 +45,7 @@
             <%= image_tag("splash-logos/BMlogo.png", class: "tenant-logo", alt: "The British Museum") %>
           <% end %>
         </div>
```

Comment:
it seems like the indentation for this file is a little wonky. can you do back and update it throughout the file?

Reply:
that might involve not using my previous suggestion, since now i see there are multiple places with indentation issues

Reply:
Hold on re-indenting everything.  Focus this commit on what you've changed.

(Mixing two commits can get a little less pleasant for reviews)

Reply:
Fixed indentations